src/asm/rgbasm.1

```diff
@@ -22,74 +22,170 @@
 .Op Fl o Ar outfile
 .Op Fl p Ar pad_value
 .Op Fl r Ar recursion_depth
-.Ar file
+.Ar
 .Sh DESCRIPTION
 The
 .Nm
-program creates an object file from an assembly source file.
+program creates an RGB object file from an assembly source file.
 The input
 .Ar file
 can be a file path, or
 .Cm \-
 denoting
 .Cm stdin .
-Its arguments are as follows:
+.Pp
+Note that options can be abbreviated as long as the abbreviation is unambiguous:
+.Fl Fl verb
+is
+.Fl Fl verbose ,
+but
+.Fl Fl ver
+could also be
+.Fl Fl version .
+The arguments are as follows:
 .Bl -tag -width Ds
-.It Fl b Ar chars
+.It Fl b , Fl Fl binary-digits Ar chars
```

Comment:
.It Fl b Ar chars , Fl Fl binary-digits Ar chars

Reply:
Interesting, the man pages I checked avoided the duplication. What's the rationale for this one?

Reply:
I’m not aware of any style guides that comment on this (BSD style guides are naturally silent on the subject of long options). My rationale is that I’ve often seen things documented as:

    .It Fl f , Fl Fl foo Ns = Ns Ar arg

And if you’re trying to use the short option, it’s not clear if there’s an equals sign or not.

Reply:
I originally did this, but it was somewhat cumbersome, having two copies of the argument spec.
All the man pages I checked used the format I used in this one.

Reply:
I don’t see how that is cumbersome. More importantly, displaying the short option without the argument is confusing (as in, it has legitimately confused me at least twice in the past). I did a quick search for manpages that do it how I suggested and (aside from several manpages I’ve written for other projects and pushed upstream) libarchive and OpenBSD sort(1) do it that way. IMO it’s easier to understand, and not burdensome.

Reply:
As a general rule, if an option requires an argument, always display it with the argument next to it; an exception can be (and generally is) made for mentions of the option in prose (i.e., not in argument lists, example invocations, etc.).

Reply:
Okay then.

Reply:
Fixed.